_posts/2017-06-27-ardublocklysenseBox.md

```diff
@@ -0,0 +1,58 @@
+
+Aktueller Stand und Funktionen
+============
+In die Oberfläche wurden Blöcke für die einzelnen Sensoren der senseBox:edu integriert. Die Sensoren können so einfach ausgelesen werden, wie es im Beispiel gezeigt wird.
+{% include image.html image=page.image1 %}
+Da der Arduino kein Display besitzt müssen anderen Ausgabemöglichkeiten genutzt werden. Hier kann der Nutzer auswählen zwischen
+- der Ausgabe auf dem seriellen Monitor (Kommandozeile) der Arduino IDE
+- dem speichern auf SD-Karte
+- direkten übertragen der Messwerte auf die openSenseMap (hierfür muss eine Station vorher registriert werden und eine Wlan oder Lan verbindung vorhanden sein).
+
+Die senseBox Blöcke sind in drei Kategorien eingeteilt:
+- den Grundlagen (mit Blöcken für LED, Piepser und der RGB LED)
+- den Sensoren (mit Blöcken für alle Sensoren die in der senseBox:edu enthalten sind)
+- verschiedenen Ausgabemöglichkeiten (SD Karte, Kommandozeile oder übertragen an die openSenseMap).
+
+
+Installation
+============
+Grundvoraussetzung für die Verwendung von Ardublockly ist, dass die [Arduino IDE](https://www.arduino.cc/en/Main/Software) installiert ist. (weitere Information auch [hier](https://edu.books.sensebox.de/de/getting_started/installation_der_software.html))
+
+Unter [Arbublockly](https://sensebox.de/blockly) befindet sich eine online Version der Oberfläche die ohne Installation auf den gängigen Browsern läuft. Hierbei muss allerdings der generierte Code mittels Copy and Paste in die Arduino IDE kopiert werden.
+
+Die offline Varianten der Ardublockly Oberfläche befinden sich im [Github Repository](github.com/senseBox/ardublockly) in den [Releases](https://github.com/sensebox/ardublockly/releases)  (die Releases werden in unregelmäßigen Abständen neu veröffentlicht). Hier gibt es für die gängigen Betriebssysteme (Windows, MacOS, Linux) eine jeweilige Version.
+Für die Offline Variante muss Python (mind. Version 2.7) installiert sein. Weitere Informationen und der Download der Software befinden sich auf der [Python Homepgage](https://www.python.org/). Unter MacOS ist Python bereits vorinstalliert. Hierbei muss nur die Ardublock.app in den Programmordner kopiert werden.
+Damit der direkte Upload oder das direkte öffnen des Sketches in der Arduino IDE funktioniert, muss der Programmpfad in den Einstellungen gesetzt werden.
+
+Damit die Sensoren ausgelesen werden können wird eine Software Bibliothek benötigt, diese beeinhaltet die Software für alle Sensoren, die in der senseBox:edu enthalten sind ([senseBox-Library](https://github.com/sensebox/senseBox_library)). Die Software Library muss heruntergeladen werden und in die Arduino IDE integriert werden (mehr Informationen dazu [hier](https://edu.books.sensebox.de/de/getting_started/installation_der_software.html)).
```

Comment:
Damit die Sensoren ausgelesen werden können, wird eine Software Bibliothek benötigt. Diese beeinhaltet die Software für alle Sensoren, die in der senseBox:edu enthalten sind.